src/Parser/Strategy/PhpExtensionStrategy.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Icanhazstring\Composer\Unused\Parser\Strategy;
+
+use PhpParser\Node;
+
+class PhpExtensionStrategy implements ParseStrategyInterface
+{
+    /** @var array<string, mixed> */
+    private $extensionConstants = [];
+
+    /** @var array<string, mixed> */
+    private $extensionFunctions = [];
+
+    /** @var array<string, mixed> */
+    private $extensionClasses = [];
+
+    public function __construct(array $extensions)
+    {
+        foreach ($extensions as $extension) {
+            $reflection = new \ReflectionExtension($extension);
+            $this->extensionConstants[$extension] = $reflection->getConstants();
+            $this->extensionFunctions[$extension] = $reflection->getFunctions();
+            $this->extensionClasses[$extension] = array_flip($reflection->getClassNames());
+        }
+    }
+
+    public function meetsCriteria(Node $node): bool
+    {
+        if ($node instanceof Node\Name\FullyQualified) {
+            foreach ($this->extensionClasses as $extensionClass) {
+                if (array_key_exists($this->getNameFromNode($node), $extensionClass)) {
+                    return true;
+                }
+            }
+        }
+        if ($node instanceof Node\Stmt\UseUse) {
+            foreach ($this->extensionClasses as $extensionClass) {
+                if (array_key_exists($this->getNameFromNode($node), $extensionClass)) {
+                    return true;
+                }
+            }
+        }
+
+        if ($node instanceof Node\Expr\ConstFetch) {
+            foreach ($this->extensionConstants as $extensionClass) {
+                if (array_key_exists($this->getNameFromNode($node), $extensionClass)) {
+                    return true;
+                }
+            }
+        }
+
+        if ($node instanceof Node\Expr\FuncCall) {
+            foreach ($this->extensionFunctions as $extensionFunction) {
+                if (array_key_exists($this->getNameFromNode($node), $extensionFunction)) {
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
+
+    public function extractNamespaces(Node $node): array
+    {
+        $searchingName = $this->getNameFromNode($node);
+
+        $matches = [];
+        foreach (
+            [
+            $this->extensionClasses,
+            $this->extensionFunctions,
+            $this->extensionConstants
+            ] as $type
+        ) {
+            foreach ($type as $phpextension => $extensionConstant) {
+                if (array_key_exists($searchingName, $extensionConstant)) {
+                    $matches[] = 'ext-' . strtolower($phpextension);
```

Comment:
```suggestion
                    $matches[] = 'ext-' . str_replace(' ', '-', strtolower($phpextension));
```

Will will result in a problem with `Zend Opcache` since its the only extension having a whitespace.
So we need to replace whitespace with a `-` so the extension to match will be `ext-zend-opcache`